app/insomnia/import-export.md

```diff
@@ -104,6 +104,13 @@ Insomnia supports the following formats:
 
 For more information on importing with the UI, go to [how to import an API spec as a document](/how-to/import-an-api-spec-as-a-document/).
 
+### Postman import {% new_in 11.5 %}
+You can migrate multiple Postman projects at once into Insomnia. To do so:
+
+1. Organize each project into its own folder.
+1. Use **Import > From Folder** from within the Insomnia UI.
+1. Insomnia will automatically detect and map the collections and environments from each folder.
+
```

Comment:
```suggestion
```
I think Postman imports have been supported for a while. I think we just need to update line 70 in this file with:

`- use_case: Import a Postman collection, OpenAPI spec, environment ({% new_in 11.5 %}) into Insomnia`